script.js

```diff
@@ -1,4 +1,72 @@
+
+// Project 1: scissors paper stone 
+// user input scissors or paper, or stone
+// input validation for "scissors" "paper" "stone"
+// assign 0,1,2 to scissors, paper, stone
+// random generate number 0,1,2
+// check input vs comPlay
+// winCondition: scissors > paper || paper > stone || stone > scissors 
+// drawCondition: input == computerRandom
+// updated for formatting 
+
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  input = input.toLowerCase();
+  var myOutputValue = "";
+  var input1 = "";
```

Comment:
You can consider renaming input1 to something that represents what the variable is going to store. In this case a better name would be computerGuess. This way, as your code gets more complicated you won't have to guess (pun intended) what the variable is storing . Remember, it's not just about getting the correct answer but ensuring accurate representation of values as well!